components/thumbnail/index.css

```diff
@@ -128,10 +128,8 @@
 			border-radius: var(--mod-thumbnail-border-radius, var(--spectrum-thumbnail-border-radius));
 		}
 	}
-	/* stylelint-enable selector-pseudo-class-no-unknown */
 
 	/* Friends should align to the top of the tabs */
```

Comment:
Probably remove this comment since it doesn't make sense without the property you removed, right?

Reply:
Yep! Just updated that ✨